app/services/crowdin_service.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "json"
+require "net/http"
+require "crowdin-api"
+
+class CrowdinService
+  # @return [Hash] of translated phrases from Crowdin
+  # Note that Crowdin doesn't have the concept of filenames like OneSky did, so we can just pull ALL phrases
+  # and since we only use phrases as needed when building XML, that's OK
+  def self.download_translated_phrases(project_id:, language_code:)
+    logger.info "Downloading translated phrases for: project id #{project_id} with language: #{language_code}"
+
+    begin
+      client
+
+      # Find the language and use its crowdin_code
+      language = Language.find_by(code: language_code)
+      raise("Can't find language #{language_code}") unless language
+      raise("Language #{language_code} has no crowdin_code") unless language.crowdin_code
+
+      # Grab export url - this is the best way I've found to get all translations for a language -AR
+      r = client.export_project_translation({targetLanguageId: language.crowdin_code, format: "android"}, nil, project_id)
```

Comment:
try the format `crowdin-json`

Reply:
there is also an `exportApprovedOnly: {bool}` option, but we aren't at a state to determine if we want that to be true or not. It should hopefully be easy to add in the future if the translator workflow starts leveraging approvals

Reply:
it looks like it's properly handling `skipUntranslatedStrings` already

Reply:
I'm going to mark this comment as resolved, the comments about `skipUntranslatedStrings` and `exportApprovedOnly` are unnecessary or not actionable currently